### DISCUSSION TITLE
Mute sound and/or hide pop-up for new messages

### DISCUSSION THREAD

**Discussion by gcaria** [cstayyab/WALC#116] [Feature Requests]
Hi there,
first of all thanks for the great work ! I really appreciate the possibility to use this app.

I'd like to know whether it is already possible to mute sound and/or hide pop-up for new messages, since I can't see this options in the settings.

**Comment by daffamumtaz2361** [comment]
Seems simple enough, I can implement it if I got the time. In the meantime, you can just use your distro's do not disturb feature (gnome and kde has it)

**Reply by gcaria** [reply]
Thanks. One downside of using "do not disturb" is that it will mute/hide everything, not just WALC

**Reply by mikebell** [reply]
I've just tested this on KDE Neon 5.21 and it hides the notification but not the sound which is odd.
